From de191f187ac47b55ebd1253dd3af7efa28c11f93 Mon Sep 17 00:00:00 2001
From: Christophe Roullier <christophe.roullier@foss.st.com>
Date: Tue, 20 Aug 2024 15:24:06 +0200
Subject: [PATCH] arm64: defconfig: put SMSC_PHY in built-in in fragment-02

Move SMSC Ethernet Phy driver from module to built-in within
fragment-02-defconfig-addons.config

Change-Id: Ie054d260520f4df7203aee1dd081716fb036e9bb
Signed-off-by: Christophe Roullier <christophe.roullier@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/399180
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
---
 arch/arm64/configs/fragment-02-defconfig-addons.config | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

--- a/arch/arm64/configs/fragment-02-defconfig-addons.config
+++ b/arch/arm64/configs/fragment-02-defconfig-addons.config
@@ -62,9 +62,9 @@ CONFIG_NVMEM_REBOOT_MODE=y
 CONFIG_NVMEM_STM32_ROMEM=y
 CONFIG_NVMEM_STM32_TAMP=y
 CONFIG_OF_OVERLAY=y
+CONFIG_PCIEASPM_POWERSAVE=y
 CONFIG_PCIE_STM32=y
 CONFIG_PCIE_STM32_EP=y
-CONFIG_PCIEASPM_POWERSAVE=y
 CONFIG_PHY_STM32_COMBOPHY=y
 CONFIG_PHY_STM32_USB2PHY=y
 CONFIG_PTM=y
@@ -78,9 +78,10 @@ CONFIG_RPMSG_IRQ=m
 CONFIG_RPMSG_TTY=m
 CONFIG_RPMSG_VIRTIO=m
 CONFIG_RTC_DRV_STM32=y
-CONFIG_SENSORS_MR75203=y
 CONFIG_SAMPLES=y
 CONFIG_SAMPLE_RPMSG_CLIENT=m
+CONFIG_SENSORS_MR75203=y
+CONFIG_SMSC_PHY=y
 CONFIG_SND_SOC_STM32_I2S=m
 CONFIG_SND_SOC_STM32_MDF=m
 CONFIG_SND_SOC_STM32_SAI=m
